README.md

```diff
@@ -28,7 +28,12 @@ volume. This is not required but is _highly_ recommended.
 
 After the data volume has been created run the server container with the named data volume:
 
-    docker run -d -p 64738:64738 -p 64738:64738/udp -v mumble-data:/etc/mumble --name mumble-server phlak/mumble
+    docker run -d \
+      -p 64738:64738 \
+      -p 64738:64738/udp \
+      -v mumble-data:/etc/mumble \
+      --name mumble-server \
+      phlak/mumble
```

Comment:
This is personal preference but I'd rather this on one line.

Reply:
done